CHANGELOG.md

```diff
@@ -12,6 +12,13 @@
 
 ### Dependency updates
 
+## [21.0.3] - 2023-05-05
```

Comment:
```suggestion
## [21.1.0] - 2023-05-08
```

I would go for a minor version, patch is only for `fixed` (you could also argue that this is fixed, not changed ofcourse)

Reply:
Don't forget to update package.json if you were to change this